Sources/Examples.swift

```diff
@@ -1288,6 +1288,42 @@ private struct Examples {
     """
 
     let organizeDeclarations = """
+    default value for `--visibilityorder`:
+    ```
+        [
+            beforeMarks,
+            instanceLifecycle,
+            open,
+            public,
+            package,
+            internal,
+            fileprivate,
+            private
+        ]
```

Comment:
Since I assume the argument itself doesn't allow the `[` and `]` characters, maybe we should omit them from the default example? 

```suggestion
        beforeMarks,
        instanceLifecycle,
        open,
        public,
        package,
        internal,
        fileprivate,
        private
```

Reply:
✅